python/lsst/dax/apdb/apdb.py

```diff
@@ -36,6 +36,7 @@
 
 import lsst.pex.config as pexConfig
 from lsst.pex.config import Field, ChoiceField, ListField
+from lsst.sphgeom import HtmPixelization, LonLat, Region, UnitVector3d
```

Comment:
It was not very well tested commit, I thought I'll just add an indication that it may not be final usable version. But it was actually OK, I can drop WIP.